capsules/src/virtual_i2c.rs

```diff
@@ -90,6 +90,7 @@ pub struct I2CDevice<'a> {
     operation: Cell<Op>,
     next: ListLink<'a, I2CDevice<'a>>,
     client: OptionalCell<&'a dyn I2CClient>,
+    modify_register_in_progess: Cell<bool>,
```

Comment:
What does this flag do?

Reply:
I separated the changes, for now the lsm303dlhc capsule is with a set only interface. As soon as we have a modify_register api figured out, I will add back the functionality.